praxis/optimizers.py

```diff
@@ -2767,16 +2753,13 @@ def _run_base_tx():
               jax.tree_map(lambda u: jnp.zeros_like(u, dtype=jnp.float32),
                            updates), emission_base_state)
 
-    # PAX makes use of vectorized map for repeated layers. XLA currently doesn't
-    # handle conds with-in vmap well and thus calls into both branches with a
-    # select. Here we rewrite a lax.cond as while_loop to get around this issue
-    # and get faster step time.
-    new_updates, new_accumulated_update, new_base_state = while_cond(
-        should_emit, _run_base_tx, [
-            jax.tree_map(jnp.zeros_like, updates), new_accumulated_update,
-            state.base_state  # pytype: disable=attribute-error  # jax-ndarray
-        ])
+    def _continue_accumulating():
+      return (jax.tree_map(jnp.zeros_like, updates), new_accumulated_update,
+              state.base_state)
```

Comment:
Can you please add   # pytype: disable=attribute-error  # jax-ndarray
Otherwise our pytype checker will complain.

Reply:
Thanks, fixed.